completions/ssh.completion.sh

```diff
@@ -5,6 +5,8 @@ _omb_module_require lib:omb-completion
 
 function _omb_completion_ssh {
   local cur
+  local additional_include_option
+  local additional_include_defined_file
```

Comment:
Can you put these declarations just before the corresponding `for` statements? The scope of the variables can be smaller in that way.

Reply:
Ok, I'll change that part

Reply:
c66312e